### PR TITLE
Add Alan as maintainer

### PR DESCRIPTION
**Pull request by cijothomas**
Alan has been an approver for several months now, and has been very active throughout the project including bug fixes, feature contributions and triaging issues etc. He has demonstrated strong understanding of the project and its direction. I'm excited to propose Alan as maintainer, so he can help with even more aspects of this project!

@CodeBlanch @SergeyKanzhelev @MikeGoldsmith

### CODE REVIEW

**Review by eddynaka — APPROVED** [review]
Well deserved! Congrats!

**Comment by reyang** [comment]
Congrats @alanwest 👍

**Review by CodeBlanch — APPROVED** [review]
Congrats @alanwest!

File: README.md
Comment (alanwest) [inline README.md:107]: Didn't notice this before. Space 👾 between New and Relic.

```suggestion
* [Alan West](https://github.com/alanwest), New Relic
```

**Review by alanwest — COMMENTED** [review]
Thanks, folks! It's a pleasure working with you all!

**Review by SergeyKanzhelev — APPROVED** [review]

**Review by bruno-garcia — APPROVED** [review]

**Comment by cijothomas** [comment]
Nominated by maintainer, with no objection from other maintainers used to be one of the pre-conditions. 
Will leave this open until end of day today, and merge.
https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer

**Pull request merged**
